turn_by_turn/iota.py

```diff
@@ -61,7 +61,7 @@ def read_tbt(file_path: Union[str, Path], hdf5_version: int = 2) -> TbtData:
             ),
         )
     ]
-    return TbtData(matrices, date, bunch_ids, nturns)
+    return TbtData(matrices=matrices, date=date, bunch_ids=bunch_ids, nturns=nturns)
```

Comment:
still here. `date` should be removed completely from IOTA

Reply:
done